streams/pm.m3u

```diff
@@ -0,0 +1,3 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="SaintPierreMiquelon1ere.pm" status="online",Saint-Pierre & Miquelon 1ere [Geo-blocked]
+https://la1ere.francetvinfo.fr/saintpierremiquelon/
```

Comment:
This is a link to the website:
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/7253922/232647427-d75b02e9-ebb1-476b-99c8-14467f44a89a.png">